catkin_home/src/navigation/base_control/src/base_odom.cpp

```diff
@@ -52,59 +50,58 @@ int main(int argc, char** argv)
   ros::Subscriber rr_enc = n.subscribe("/base_control/back/encoders", 100, reCallBack);
   tf::TransformBroadcaster odom_broadcaster;
   
-  //initialize displacement variables
+  // Initialize displacement variables.
   double x = 0.0;
   double y = 0.0;
   double th = 0.0;
 
-  //initialize velocity variables
+  // Initialize velocity variables.
   double vx = 0.0;
   double vy = 0.0;
   double vth = 0.0;
   
   ros::Time current_time = ros::Time::now();
   
-  //checks the loop at a rate of 25Hz
+  // Checks the loop at a rate of 25Hz -> 0.04 seconds.
   ros::Rate r(25.0);
```

Comment:
Make a constant

Reply:
Making [ros::Rate r] a constant ,throw an error in catkin_make process.